docker/python3/Dockerfile

```
@@ -1,5 +1,4 @@
-FROM nvidia/cuda:11.2.1-cudnn8-devel
-LABEL maintainer="CuPy Team"
+FROM nvidia/cuda:11.1-cudnn7-devel
```

```suggestion
FROM nvidia/cuda:11.1.1-cudnn8-devel
```

This was my mistake, the image name in the original Dockerfile was wrong.
https://hub.docker.com/layers/nvidia/cuda/11.1.1-cudnn8-devel/images/sha256-a47bd681dcfcdaa2f8bd35a1646875418ebc966a414d402ed8ca068137ff2e04?context=explore